README.md

```diff
@@ -1,24 +1,48 @@
-# README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+# Kitty News API
+
+## Description
+This is the API to our project Kitty News. Here we are authenticating users, storing articles, saving all our data and also making calls to external APIs such as Stripe. This is built with Ruby on Rails. We are using Amazon Web Services to store our images.
+- User registration
+- User authentication(Log-in/Log-off)
+- Yearly subscription plan
+- Display articles in a category
+
+## The code
+Deployed App: (https://kitty-news.herokuapp.com/) </br>
```

Comment:
We can delete this line as we said before :)

Reply:
The heroku deployed address is not showing any where else so lets keep this!